csharp/avro/src/KafkaAvroDeserializer.cs

```diff
@@ -39,7 +39,21 @@ public T Deserialize(ReadOnlySpan<byte> data, bool isNull, SerializationContext
                 return default(T);
             }
 
-            return (T) this.serializer.Deserialize(new MemoryStream(data.ToArray()), typeof(T), CancellationToken.None);
+            BinaryContent content = new BinaryContent
```

Comment:
Just double check on this - I see that in latest github [here](https://github.com/Azure/azure-sdk-for-net/blob/d00e82a801a1d4c7d32658ea16d8bca5e8d71c11/sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/src/SchemaRegistryAvroSerializer.cs#L376) the signature assume we will pass in MessageContent. are we using BinaryContent here because of the SDK version, or something else force us to use BinaryContent here?

Reply:
This uses beta.7 of the SDK to keep the preamble backward compatibility. In that version, the [Deserialize](https://github.com/Azure/azure-sdk-for-net/blob/05f426a4ebc1e3d448b51c34f65321e0f46491fe/sdk/schemaregistry/Microsoft.Azure.Data.SchemaRegistry.ApacheAvro/src/SchemaRegistryAvroSerializer.cs#L272) method takes in BinaryContent, which was renamed to MessageContent in later beta versions.